tests/conftest.py

```diff
@@ -589,97 +598,85 @@ def deposit(app, db, user, pdf_file, bucket_location, deposit_json):
-        'startDate':
-        '2019-01-01',
-        'endDate':
-        '2020-01-01',
-        'identifiedBy': {
-            'type': 'bf:Local',
-            'source': 'RERO',
-            'value': '1111'
-        },
-        'investigators': [{
-            'agent': {
-                'preferred_name': 'John Doe'
-            },
-            'role': ['investigator'],
-            'affiliation': 'IST',
+        'metadata': {
+            'name':
+            'Project 1',
+            'description':
+            'Description of the project',
+            'startDate':
+            '2019-01-01',
+            'endDate':
+            '2020-01-01',
             'identifiedBy': {
                 'type': 'bf:Local',
                 'source': 'RERO',
-                'value': '2222'
-            }
-        }],
-        'funding_organisations': [{
-            'agent': {
-                'preferred_name': 'Funding organisation'
+                'value': '1111'
             },
-            'identifiedBy': {
-                'type': 'bf:Local',
-                'source': 'RERO',
-                'value': '3333'
-            }
-        }]
+            'investigators': [{
+                'agent': {
+                    'preferred_name': 'John Doe'
+                },
+                'role': ['investigator'],
+                'affiliation': 'IST',
+                'identifiedBy': {
+                    'type': 'bf:Local',
+                    'source': 'RERO',
+                    'value': '2222'
+                }
+            }],
+            'funding_organisations': [{
+                'agent': {
+                    'preferred_name': 'Funding organisation'
+                },
+                'identifiedBy': {
+                    'type': 'bf:Local',
+                    'source': 'RERO',
+                    'value': '3333'
+                }
+            }]
+        }
     }
 
 
 @pytest.fixture()
-def make_project(db, project_json, make_user):
+def make_project(db, project_json, make_user, identity_simple):
     """Factory for creating project."""
 
     def _make_project(role='submitter', organisation=None):
         user = make_user(role, organisation)
 
-        project_json['user'] = {
+        project_json['metadata']['user'] = {
             '$ref': 'https://sonar.ch/api/users/{pid}'.format(pid=user['pid'])
         }
 
-        project_json['organisation'] = {
+        project_json['metadata']['organisation'] = {
             '$ref':
             'https://sonar.ch/api/organisations/{pid}'.format(pid=organisation)
         }
 
-        project_json.pop('pid', None)
-
-        record = ProjectRecord.create(project_json,
-                                      dbcommit=True,
-                                      with_bucket=False)
-        record.commit()
-        record.reindex()
-        db.session.commit()
+        project_json.pop('id', None)
 
-        return record
+        return sonar.resources['projects'].service.create(
```

Comment:
Why not redefine `create` in the SONARRecord class to add a default value for identity?